lightning/src/util/wakers.rs

```diff
@@ -254,6 +255,21 @@ impl Sleeper {
 			vec![Arc::clone(&fut_a.state), Arc::clone(&fut_b.state), Arc::clone(&fut_c.state)];
 		Self { notifiers }
 	}
+	/// Constructs a new sleeper from four futures, allowing blocking on all three at once.
```

Comment:
Nit: The comment says "allowing blocking on all three at once," but the function takes four futures.